src/routes/routes.service.ts

```diff
@@ -1,4 +1,17 @@
 import { Injectable } from '@nestjs/common';
+import { CreateRouteDto } from './dto/create-route.dto';
+import { routeRepository } from './routes.repository';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Route } from './entities/route.entity';
```

Comment:
```suggestion
```

Analysis failed due to unused import. Resolving this will render the checks green 👌